docs/standard/microservices-architecture/docker-application-development-process/docker-app-development-workflow.md

```diff
@@ -386,7 +386,7 @@ In addition, you need to perform step 2 (adding Docker support to your projects)
 
 ## Using PowerShell commands in a Dockerfile to set up Windows Containers 
 
-[Windows Containers](https://msdn.microsoft.com/en-us/virtualization/windowscontainers/about/about_overview) allow you to convert your existing Windows applications into Docker images and deploy them with the same tools as the rest of the Docker ecosystem. To use Windows Containers, you run PowerShell commands in the Dockerfile, as shown in the following example:
+[Windows Containers](https://msdn.microsoft.com/virtualization/windowscontainers/about/about_overview) allow you to convert your existing Windows applications into Docker images and deploy them with the same tools as the rest of the Docker ecosystem. To use Windows Containers, you run PowerShell commands in the Dockerfile, as shown in the following example:
```

Comment:
change this to https://docs.microsoft.com/virtualization/windowscontainers/about/index